ci/common/build.sh

```diff
@@ -7,27 +7,26 @@ build_make() {
 }
 
 build_deps() {
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
     DEPS_CMAKE_FLAGS="${DEPS_CMAKE_FLAGS} ${CMAKE_FLAGS_32BIT}"
   fi
-  if [[ "${FUNCTIONALTEST}" == "functionaltest-lua" ]]; then
+  if test "x${FUNCTIONALTEST}" = "xfunctionaltest-lua" ; then
```

Comment:
BTW, `/usr/bin/\[ --help` prints usage, starting with `Usage: test EXPRESSION` (note: `test`). `test --help`  prints nothing (guess because it is a test for emptyness, like `[ --help ]` would be). And executables have different sizes, `[` is larger by 4 KiB.